src/textarea/characterLimitChecker.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+
+interface Props {
+    radius: number;
+    stroke: string;
+    strokeDasharray: string;
+    limit: number;
+}
+
+interface State {}
+
+class CharacterLimitChecker extends React.Component<Props, State> {
+    private CO_ORDINATE: string = '50%';
+
+    render() {
+        return (
+            <svg id="character-limit-circle" data-qa={'character-limit-circle'} className={'character-limit-circle'}>
+                <circle id="gray" cx={this.CO_ORDINATE} cy={this.CO_ORDINATE} r={this.props.radius} />
+                <circle
+                    id="colored-circle"
+                    data-qa={'colored-circle'}
+                    cx={this.CO_ORDINATE}
+                    cy={this.CO_ORDINATE}
+                    r={this.props.radius}
+                    className={this.props.stroke}
+                    style={{ strokeDasharray: this.props.strokeDasharray }}
+                />
+                <text
```

Comment:
This is not a valid HTML tag

Reply:
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-dasharray this applies to svg if you mean strokeDashArray

Reply:
i mean <text>

Reply:
<text> can be used inside <svg> tag https://developer.mozilla.org/en-US/docs/Web/SVG/Element/text